vagrant/README.md

```diff
@@ -0,0 +1,29 @@
+# Vagrant
+
+Here you will find a `Vagrantfile` to build a development environment for `kvm-vmi`.
+
+# Requirements
+
+- `vagrant`
+- `vagrant-libvirt` plugin
+- `ansible >= 2.2.1.0` (use a virtualenv, install it via pip)
```

Comment:
It's not necessary.
The only requirement is to have a version of `Ansible >= 2.2.1.0`.
But even on Fedora 23, my `ansible` was only `2.2.0.0` which was not enough to workaround a bug in the `git` module.